tests/js/spec/views/organizationDiscover/queryBuilder.spec.jsx

```diff
@@ -9,7 +9,7 @@ describe('Query Builder', function() {
     const external = queryBuilder.getExternal();
 
     expect(external.projects).toEqual([2]);
-    expect(external.fields).toEqual(['event_id', 'timestamp']);
+    expect(external.fields).toHaveLength(47);
```

Comment:
Why are we just testing length here, is it because `fields` is an array of 47 strings now?

It makes it harder for readers to see what to expect for `external.fields` (i.e. it could also be a 47 char string). If we don't care what's in `fields`, could we do something like `toEqual(expect.arrayContaining(expect.any(String)))` or if we semi-care what gets added/removed from fields, maybe just snapshot it.